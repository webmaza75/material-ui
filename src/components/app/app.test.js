import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import App from './app.jsx';

describe(`App`, () => {
  it(`App renders correctly`, () => {
    const renderer = new ShallowRenderer();
    const result = renderer
      .render(<App />);

    expect(result).toMatchSnapshot();
  });
});
