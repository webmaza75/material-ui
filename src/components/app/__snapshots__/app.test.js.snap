// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App renders correctly 1`] = `
<CssBaseline>
  <Main />
</CssBaseline>
`;
